cmake/AwsTestHarness.cmake

```diff
@@ -32,12 +32,14 @@ function(generate_test_driver driver_exe_name)
     target_link_libraries(${driver_exe_name} PRIVATE ${CMAKE_PROJECT_NAME})
     set_target_properties(${driver_exe_name} PROPERTIES LINKER_LANGUAGE C C_STANDARD 99)
     target_compile_definitions(${driver_exe_name} PRIVATE AWS_UNSTABLE_TESTING_API=1)
-    target_include_directories(${driver_exe_name} PRIVATE ${CMAKE_CURRENT_LIST_DIR})
+    target_include_directories(${driver_exe_name} PRIVATE ${CMAKE_CURRENT_LIST_DIR} ${CMAKE_CURRENT_BINARY_DIR}/generated-include)
+
+    configure_file("${CMAKE_CURRENT_SOURCE_DIR}/test_crosscheck.inl.template" "${CMAKE_CURRENT_BINARY_DIR}/generated-include/test_crosscheck.inl")
```

Comment:
I think this will cause other repos without this file to break. Can you put it in include/aws/testing so it gets installed?